pkgs/top-level/all-packages.nix

```diff
@@ -3941,6 +3941,7 @@ in
   lolcat = callPackage ../tools/misc/lolcat { };
 
   lsd = callPackage ../tools/misc/lsd { };
+  pazi = callPackage ../tools/misc/pazi { };
```

Comment:
Please keep it looking nice and add an empty line next before, if it's not related to `lsd`. :)

Reply:
Done. Well I also moved it at the right place (I think) to keep it sorted.